runtime/ops/net.rs

```diff
@@ -47,6 +54,7 @@ pub fn init(rt: &mut deno_core::JsRuntime) {
   super::reg_json_sync(rt, "op_listen", op_listen);
   super::reg_json_async(rt, "op_datagram_receive", op_datagram_receive);
   super::reg_json_async(rt, "op_datagram_send", op_datagram_send);
+  super::reg_json_async(rt, "op_resolve_addr", op_resolve_addr);
```

Comment:
I feel like there's more appropriate name... like `op_dns_resolve`, because `op_resolve_addr` is a bit ambiguous.
 Anyone have thoughts on this?

Reply:
@magurotuna `op_dns_resolve` sounds good

Reply:
Okay, done :)